### PR TITLE
chore(patients): split patient address field into subfields

### LINKED ISSUE #2141: Split patient address field into subfields
## 🚀  Split patient address field into subfields

Patients' addresses should be split into appropriate subfields, such as Street, Town/City, Region and Postal code.

## Motivation

Currently, the address field of Patient contact information is stored as a single string of no specific format. This has several potential problems:
1. Possibly leads to confusion and/or duplication if the same address is entered in various formats for different patients at the same address.  Administrative errors have potentially serious ramifications in a clinical context.
2. Database actions such as sorting and filtering by address will presumably need individual fields and some standard formatting in order to be practically useful.
3. Specific address info can't easily be pulled into other areas, e.g. - displaying zipcode alongside patient name in search field results.

Screen Shot 2020-06-18 at 20 06 22

## Example

The Patient address should be entered, stored and returned as separate subfields:

Street Address
Street Address 2 (optional)
Town/City
Zip/Postcode
Country (can be set to the hospital's location country by default, but useful to have the field if international patients are a possibility)

Some fields could potentially benefit from setting standard formats, but I think that can be raised as a separate issue as needed.

### PR DESCRIPTION
**Pull request by thelderop**
Fix #2141 

Relevant files changed:

GeneralInformation.tsx
Patient.ts
index.ts
patient-slice.ts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -145,7 +145,7 @@
   "husky": {
     "hooks": {
       "pre-commit": "npm run lint-staged",
-      "commit-msg": "npm run commitlint -- -E HUSKY_GIT_PARAMS"
+      "commit-msg": "npm run commitlint -E HUSKY_GIT_PARAMS"
```
Comment (tehKapa) [inline package.json:148]: Why you change this?

Comment (thelderop) [reply package.json:148]: Oh sorry- I had some issues with git, and I changed that line while troubleshooting.

Comment (tehKapa) [reply package.json:147]: You must also sign our CLA. Thanks

**Comment by jackcmeyer** [comment]
Hi @thelderop. Before we can review this, can you please make sure all of the PR checks are passing! thanks!

**Pull request closed without merging**